aten/src/ATen/test/dlconvertor_test.cpp

```diff
@@ -30,3 +30,15 @@ TEST(TestDlconvertor, TestDlconvertorNoStrides) {
 
   ASSERT_TRUE(a.equal(b));
 }
+
+TEST(TestDlconvertor, TestDlconvertorNullDeleter) {
+  manual_seed(123);
+
+  Tensor a = rand({3, 4});
+  DLManagedTensor* dlMTensor = toDLPack(a);
+  dlMTensor->deleter = nullptr;
```

Comment:
This doesn't look right. What you have written here effectively says, "Allocate a DLPack tensor from PyTorch, and then *leak* the memory associated with it by forgetting the deleter." While I agree it exercises the code, this isn't code you'd ever want to actually see in production. Your best bet is probably to manually allocate a DLPack tensor with no deleter; you can look at toDLPack for an idea of what fields you need to fill.